tests/ui/zombie_processes.rs

```diff
@@ -0,0 +1,87 @@
+#![warn(clippy::zombie_processes)]
+
+use std::process::{Child, Command};
+
+fn main() {
+    {
+        let mut x = Command::new("").spawn().unwrap();
```

Comment:
Could you add one test with `#[allow]` on the command creation to ensure that it can be correctly suppressed at this node? I'm pretty sure it will be, but it's better to check this explicitly :)